src/Collections-Support/CharacterSet.class.st

```diff
@@ -142,12 +136,13 @@ CharacterSet >> hash [
 	^self byteArrayMap hash
 ]
 
-{ #category : #'collection ops' }
+{ #category : #testing }
 CharacterSet >> includes: aCharacter [
-	aCharacter asciiValue >= 256
+
+	^aCharacter asciiValue >= 256
 		ifTrue: ["Guard against wide characters"
-			^false].
-	^(map at: aCharacter asciiValue + 1) > 0
+					^false]
```

Comment:
This is no longer a guard, but a regular if/else; so the extra return is not needed.

Reply:
Yes - it is not a guard using an early return anymore. Nonetheless the way it is written protects from invalid accessing of the map - so functionality is given

Reply:
Hi Torsten,

Can you explain how the extra return prevents invalid access to the map?  If it wasn't there, how could a wide character access the map?

Thanks,
Alistair

Reply:
I thought you questioned the transformation itself. 

s := CharacterSet newFrom: 'abc'.
s includes: 300 asCharacter 

Now I see that you meant the second ^ left on the true condition. 
This is just a leftover and not needed.

Reply:
Yes. Sorry, I should've been clearer.